README.md

```diff
@@ -19,6 +19,11 @@ $ cargo install heretek --locked
 
 Or download from [github releases](https://github.com/wcampbell0x2a/heretek/releases).
 
+### Build Targets
+`clap-derivie` requires the `linux-musl` target.
+
+Install it with `rustup target add x86_64-unknown-linux-musl`
```

Comment:
This isn't a problem with just `clap-derive`, as the final binary that I generated need to be linked with a libc. I just so happen to pick `musl`.

I could just remove the `musl` target in the `.cargo/config`. I really only use it to build static binaries that are easy to distribute. `gnu` is fine for most people, and is the default host already installed for most `x86` systems that aren't Alpine. I think I'd rather do that.

Reply:
Ok so it probably could just be a none issue and not needed as documentation at all. I'll close the MR.